crates/bevy_openxr/src/conversion.rs

```diff
@@ -0,0 +1,27 @@
+use bevy_xr::{XrDuration, XrTime};
+use glam::{Quat, Vec2, Vec3};
```

Comment:
Instead of directly using glam, we should probably use the re-exports from `bevy_math`

Reply:
Thanks, I will fix it.